maintainers/maintainer-list.nix

```diff
@@ -3894,6 +3894,12 @@
     githubId = 631802;
     keys = [ { fingerprint = "099E 3F97 FA08 3D47 8C75  EBEC E0EB AD78 F019 0BD9"; } ];
   };
+  chipsenkbeil = {
+    email = "nyxar49@gmail.com";
```

Comment:
Why are these all your email?

Also: these seem to be the maintainers of the upstream project. You should only add yourself as a maintainer in this case.

Reply:
thanks for clarification, i've removed them